persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -42,6 +42,10 @@ CREATE INDEX message_status_index ON Message(status);
 CREATE INDEX message_expire_time_index ON Message(expire_after_millis);
 CREATE INDEX message_self_deletion_end_date ON Message(self_deletion_end_date);
 
+CREATE INDEX idx_msg_conv_date_desc ON Message(conversation_id, creation_date DESC);
+CREATE INDEX idx_msg_conv_vis ON Message(conversation_id, visibility);
+CREATE INDEX idx_msg_conv_vis_date_desc ON Message(conversation_id, visibility, creation_date DESC);
```

Comment:
you don't need the 3, we can drop idx_msg_conv_date_desc and idx_msg_conv_vis the idx_msg_conv_vis_date_desc will do the same trick, unless we have a query that only care for creation date